### PR TITLE
NFT-244: Monitor email pipeline with metrics

### PR DESCRIPTION
**Pull request by adamgobes**
Sends an email every 24 hours with the following:
monitoring

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -19,4 +19,5 @@ DATABASE_URL="postgresql://user:password@localhost:5432/notifications?schema=pub
 NEXT_PUBLIC_PAWN_SHOP_API_URL="https://nft-pawn-shop-rinkeby.vercel.app"
 NEXT_PUBLIC_NOTIFICATIONS_FREQUENCY_HOURS=1
 EVENTS_API_SECRET_KEY="very secret key"
-BACKED_NOTIFICATIONS_EMAIL_ADDRESS="no-reply@backed.xyz"
\ No newline at end of file
+BACKED_NOTIFICATIONS_EMAIL_ADDRESS="no-reply@backed.xyz"
+EMAIL_MONITOR_REPORT_TO="adamgobes@gmail.com"
```
Comment (wilsoncusack) [inline .env.test:23]: @adamgobes are you sure you want this public? maybe make this something dummy?

Comment (cnasc) [reply .env.test:23]: probably good to make it dummy, but it is also in the log for every commit adam's made

File: pages/api/events/cron/monitorEmailSends.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  let configs: Config[];
+  if (process.env.VERCEL_ENV === 'production') {
+    configs = prodConfigs;
+  } else {
+    configs = devConfigs;
+  }
+
+  try {
+    const currentTimestamp = Math.floor(new Date().getTime() / 1000);
+    const dayAgo = currentTimestamp - 24 * 3600;
```
[the remark this answers is not in the document]
Comment (adamgobes) [reply pages/api/events/cron/monitorEmailSends.ts:38]: nope, but can add in a future PR

File: .env.test (hunk 1 above)
Comment (adamgobes) [reply .env.test:23]: will do in a follow up

**Pull request merged**